libs/blocks/text/text.css

```diff
@@ -120,7 +120,7 @@
   align-items: center;
 }
 
-.text-block .icon-area {
+.text-block .icon-area:not(:has(span.icon)) {
```

Comment:
This does seem to work, but to me it sounds a bit odd that we're targeting an icon area without icons. Can we track down the original reasoning for why this was needed? I don't have a use-case where this wouldn't work in mind, but I'd suggest caution when QA-ing this.